### PR TITLE
Altcoin integration

### PR DESCRIPTION
**Pull request by peerchemist**
My take on: https://github.com/chainside/btcpy/pull/18

### CODE REVIEW

File: btcpy/lib/codecs.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
 from base58 import b58encode_check, b58decode_check
 
 from .bech32 import decode, encode
-from ..setup import is_mainnet, net_name
+from ..setup import net_name
+from ..setup import NETWORKS
```
Comment (oskyk) [inline btcpy/lib/codecs.py]: You can merge this 2 lines together same in crypto.py

**Comment by oskyk** [comment]
Pretty good way to solve this problem.

**Comment by peerchemist** [comment]
@SimoneBronzini any comments on this?

**Comment by SimoneBronzini** [comment]
@peerchemist really sorry about the delay, will take a look at it as soon as possible. The next days might be a good time.

**Pull request closed without merging**